frontend/src/features/admin-form/settings/SettingsEmailsPage.tsx

```diff
@@ -0,0 +1,44 @@
+import { FormControl, Skeleton } from '@chakra-ui/react'
+
+import { FormResponseMode } from '~shared/types'
+
+import FormLabel from '~components/FormControl/FormLabel'
+import { TagInput } from '~components/TagInput'
+
+import { CategoryHeader } from './components/CategoryHeader'
+import { EmailFormSection } from './components/EmailFormSection'
+import { useAdminFormSettings } from './queries'
+
+const AdminEmailSection = (): JSX.Element => {
+  const { data: settings } = useAdminFormSettings()
+
+  const isEmailOrStorageMode =
+    settings?.responseMode === FormResponseMode.Email ||
+    settings?.responseMode === FormResponseMode.Encrypt
+
+  if (settings && isEmailOrStorageMode) {
+    return <EmailFormSection settings={settings} />
+  } else {
+    return <EmailFormSectionSkeleton />
+  }
+}
+
+export const EmailFormSectionSkeleton = (): JSX.Element => {
```

Comment:
1. `EmailFormSectionSkeleton` doesn't seem to be used outside of this file. We can omit the export.
2. We usually avoid exporting multiple react components from the same file.

Reply:
Gotcha!